.github/workflows/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.yml]
+indent_size = 2
```

Comment:
This is a matter of clear personal preference.. I do prefer 4 spaces..

Reply:
I do too, but YAML is aweful with mixed two and four spaces in lists.